java/mykola-melnyk/src/com/zoolatech/drafts/Task2.java

```diff
@@ -0,0 +1,159 @@
+package com.zoolatech.drafts;
+
+import java.util.Scanner;
+
+public class Task2 {
```

Comment:
I'm expecting that every student will provide its own solution: https://github.com/zoolacode/zoola-academy-homework/blob/c021f3d638211c9bf498f4e41f733ec4e3232313/java/Anton-Kozyk/src/com/zoolatech/lecture2/tasks/_2/Task2.java

Reply:
Resolved: rewrote the program code completely.